lib/List/index.js

```diff
@@ -1,27 +1,35 @@
 import React from 'react';
+import cx from 'classnames';
 import PropTypes from 'prop-types';
 import ListHead from './ListHead';
 
-const List = props => (
-  <div className="list">
-    <ListHead
-      title={props.title}
-      action={props.action}
-    />
+const List = (props) => {
+  const listClasses = cx({
+    'list--bordered-right': props.borderedRight,
+  });
 
-    <div className="list__body">
-      { React.Children.map(props.children, child => (
-        <div className="list__row">
-          { React.cloneElement(child, {}) }
-        </div>
-      )) }
+  return (
+    <div className={`list ${listClasses}`}>
```

Comment:
Maybe this would be a good opportunity to use `cx(list, {'list--bordered-right': props.borderedRight})` again?